test/e2e/workflow_nowebhooks_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2022 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/kiegroup/kogito-serverless-operator/test"
+	"github.com/kiegroup/kogito-serverless-operator/test/utils"
+
+	//nolint:golint
+	//nolint:revive
+	. "github.com/onsi/ginkgo/v2"
+
+	//nolint:golint
+	//nolint:revive
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("SonataFlow Operator - no webhooks", Serial, func() {
+
+	Describe("ensure that Operator and Operand(s) can run in restricted namespaces - no webhooks", Ordered, func() {
+		It("Prepare the environment - no webhooks", func() {
+			var controllerPodName string
+			operatorImageName, err := utils.GetOperatorImageName()
+			ExpectWithOffset(1, err).NotTo(HaveOccurred())
+
+			By("installing CRDs")
+			cmd := exec.Command("make", "install-no-webhooks")
```

Comment:
This step shouldn't be needed as this will call just
`$(KUSTOMIZE) build config/crd-no-webhooks | kubectl apply -f -`
which is a subset of what `make deploy-no-webhooks` calls, i.e.
`$(KUSTOMIZE) build config/default-no-webhooks | kubectl apply -f -`

Reply:
It is just a copy from the existing test.

Reply:
removed, let's see how it goes.